homeassistant/components/logbook.py

```diff
@@ -239,6 +264,24 @@ def humanify(events):
                     entity_id)
 
 
+def _get_excludes(config):
+    """Get lists of excluded entities and platforms."""
+    exclude_entities = []
+    exclude_platforms = []
+    exclude = config[DOMAIN].get(CONF_EXCLUDE) if DOMAIN in config else None
+    if exclude:
+        exclude_entities = exclude.get(CONF_ENTITIES) if \
+            CONF_ENTITIES in exclude else []
+        exclude_platforms = exclude.get(CONF_PLATFORMS) if \
+            CONF_PLATFORMS in exclude else []
+    customize = config[HA_DOMAIN].get(CONF_CUSTOMIZE)
```

Comment:
Instead of looking at `customize`, look for the hidden attribute on the state change events.